crates/tools/gnullvm/readme.md

```diff
@@ -0,0 +1,16 @@
+The Windows umbrella lib (targeting mingw-w64 LLVM tooling) is generated using the following steps:
+
+0. Ensure MSYS2 MinGW environment is installed (https://www.mingw-w64.org/downloads/)
+1. Ensure `~\.cargo\config` has the following block to override toolchain defaults (customize paths as needed):
+
+    ```toml
+    [target.x86_64-pc-windows-gnu]
+    linker = "C:\\msys64\\mingw64\\bin\\x86_64-w64-mingw32-gcc.exe"
+    ar = "C:\\msys64\\mingw64\\bin\\ar.exe"
+    ```
+
+2. Open `MSYS2 MinGW 64-bit`
+3. Execute: `pacman -Syuu --noconfirm` (repeat until no further updates available)
+4. Execute `pacman --needed -S mingw-w64-x86_64-llvm`
```

Comment:
Should probably add `mingw-w64-x86_64-clang`. do we need `mingw-w64-clang-x86_64-toolchain` as well?

```suggestion
4. Execute `pacman --needed -S mingw-w64-x86_64-llvm`
```

Reply:
`mingw-w64-clang-x86_64-toolchain` is a bit overkill: https://packages.msys2.org/group/mingw-w64-clang-x86_64-toolchain as it installs everything that might be required for simple C/C++ projects (including Clang and libunwind).

If you want I can find out minimal required set of packages, probably it'd be: `pacman --needed -S mingw-w64-x86_64-clang mingw-w64-x86_64-libunwind`